e2e/tests/mcp-setup-wizard.spec.ts

```
@@ -0,0 +1,330 @@
+  await page.locator('#listener-name').fill(name);
+  await page.locator('#listener-port').clear();
+  await page.locator('#listener-port').fill(port);
+
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.locator('#listener-protocol').selectOption(protocol);
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Next' }).click();
+
+  const addButton = page
+    .locator('.pf-v6-c-modal-box')
+    .getByRole('button', { name: 'Add' });
```

ESLint

[error] Replace `⏎····.locator('.pf-v6-c-modal-box')⏎····` with `.locator('.pf-v6-c-modal-box')`

(prettier/prettier)